e2e/wdio/platform/tests/radio-button-group.e2e-spec.ts

```diff
@@ -0,0 +1,436 @@
+        click(radioButtonGroupPage.radioButtonLabelByName(radioButtonWithDisabledButtonAndValidationError));
+        expect(getText(selectedValueLabel, radioButtonWithDisabledButtonAndValidationError)).toContain(januaryValue);
+        expect(getAttributeByName(radioButtonGroupPage.radioButtonInputByName(radioButtonWithDisabledButtonAndValidationError)
+            , 'aria-checked')).toBe('true');
+        click(radioButtonGroupPage.actionButtonByName(resetDisabledValidationGroupButtonIndex));
+        expect(monthNames).not.toContain(getText(selectedValueLabel, radioButtonWithDisabledButtonAndValidationError));
+    });
+
+
+    it('Verify radio group using FormGroup and FormControl', () => {
+        scrollIntoView(radioButtonGroupPage.radioButtonInputByName(radioButtonUsingFormGroupAndFormControl));
+        click(radioButtonGroupPage.radioButtonLabelByName(radioButtonUsingFormGroupAndFormControl), marchIndex);
+        expect(getText(selectedValueLabel, radioButtonUsingFormGroupAndFormControl)).toContain(marchValue);
+        expect(getAttributeByName(radioButtonGroupPage.radioButtonInputByName(radioButtonUsingFormGroupAndFormControl)
+            , 'ng-reflect-is-disabled')).toBe('true');
+        expect(getAttributeByName(radioButtonGroupPage.radioButtonInputByName(radioButtonUsingFormGroupAndFormControl)
+            , 'aria-checked', marchIndex)).toBe('true');
+    });
+
+    it('Verify inline compact radio group with default selection', () => {
+        scrollIntoView(radioButtonGroupPage.radioButtonInputByName(radioButtonInlineWithDefaultSelection));
+        click(radioButtonGroupPage.radioButtonLabelByName(radioButtonInlineWithDefaultSelection));
+        expect(getText(selectedValueLabel, radioButtonInlineWithDefaultSelection)).toContain(winterValue);
+        expect(getAttributeByName(radioButtonGroupPage.radioButtonInputByName(radioButtonInlineWithDefaultSelection)
+            , 'aria-checked')).toBe('true');
+
+    });
+
+    it('Verify inline compact radio group', () => {
+        scrollIntoView(radioButtonGroupPage.radioButtonInputByName(radioButtonInlineCompact));
+        click(radioButtonGroupPage.radioButtonLabelByName(radioButtonInlineCompact));
+        expect(getText(selectedValueLabel, radioButtonInlineCompact)).toContain(winterValue);
+        expect(getAttributeByName(radioButtonGroupPage.radioButtonInputByName(radioButtonInlineCompact)
+            , 'aria-checked')).toBe('true');
+    });
+
+    it('Verify radio group using FormGroup and FormControl', () => {
+        scrollIntoView(radioButtonGroupPage.radioButtonInputByName(radioButtonWithDisabledItem));
+        click(radioButtonGroupPage.radioButtonLabelByName(radioButtonWithDisabledItem), marchIndex);
+        expect(getText(selectedValueLabel, radioButtonWithDisabledItem)).toContain(marchValue);
+        expect(getAttributeByName(radioButtonGroupPage.radioButtonInputByName(radioButtonWithDisabledItem)
+            , 'ng-reflect-is-disabled')).toBe('true');
```

Comment:
you use "aria-checked" and "ng-reflect-is-disabled" many times. Might be good to create variables for them in the appData file in case these change.